kata-webhook/README.md

```diff
@@ -27,7 +30,15 @@ by default, except for workloads we know for sure will not work with Kata, use
 the [admission webhook](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#admission-webhooks)
 and sample admission controller we created by running
 
+> **Note:**
+> By default, the RuntimeClass name used in this webhook is `kata`. If your
+> cluster is configured with another RuntimeClass, you'll need to change the
+> `RUNTIME_CLASS` to the correct value in the [webhook.yaml](deploy/webhook.yaml#L27).
```

Comment:
`RUNTIME_CLASS` .... variable ? dictionary ?

Reply:
`deploy/webhook.yaml#L27` is too specific, line number can change at any moment

Reply:
Thanks @devimc, addressed both comments